WolvenKit.App/ViewModels/Dialogs/SoundModdingViewModel.cs

```diff
@@ -0,0 +1,220 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Prism.Commands;
+using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
+using Splat;
+using WolvenKit.Common.Services;
+using WolvenKit.Functionality.Services;
+using WolvenKit.Models;
+using WolvenKit.Modkit.RED4.Sounds;
+using WolvenKit.ProjectManagement.Project;
+
+namespace WolvenKit.ViewModels.Dialogs
+{
+    public class SoundModdingViewModel : DialogViewModel
+    {
+        private readonly ISettingsManager _settings;
+        private readonly ILoggerService _logger;
+        private readonly IProjectManager _projectManager;
+
+        private readonly SoundEventMetadata _metadata;
+        private readonly ModInfo _info;
+
+        private readonly JsonSerializerOptions _options = new()
+        {
+            WriteIndented = true,
+            DefaultIgnoreCondition = System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull,
+            PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+            IgnoreReadOnlyProperties = true,
+        };
+
+        public SoundModdingViewModel()
+        {
+            _settings = Locator.Current.GetService<ISettingsManager>();
+            _logger = Locator.Current.GetService<ILoggerService>();
+            _projectManager = Locator.Current.GetService<IProjectManager>();
+
+            _metadata = new SoundEventMetadata();
+
+            SaveCommand = ReactiveCommand.Create(() => Save());
+            CancelCommand = ReactiveCommand.Create(() => FileHandler(null));
+
+            AddCommand = new DelegateCommand(AddEvents, CanAddEvents);
+
+            // load events
```

Comment:
I think this ctor is doing too much from here on. For example I would split it in multiple funcs, e.g.: `LoadSoundEvents`, `LoadInfo`, `PopulateFiles`.

Might also be nice to have them moved out of ctor (in a callback event or something), but I'm not that familiar with WolvenKit architecture to say where.